### PR TITLE
Added telemetry

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
Using https://github.com/Drakkar-Software/Metrics-Server

image

### CODE REVIEW

File: core/octobot.py [hunk 1]
```diff
@@ -47,6 +47,9 @@ def __init__(self, config, ignore_config=False, reset_trading_history=False):
             BOT_TOOLS_RECORDER: None,
         }
 
+        # metrics if enabled
+        self.metrics_handler = None
```
Comment (Werdor) [inline core/octobot.py]: Why not in Initializer ?

Comment (GuillaumeDSM) [reply core/octobot.py]: To me it seems more legit to store it in octobot

File: tools/metrics_manager.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        first_evaluator = next(iter(self.octobot.get_symbols_tasks_manager().values())).get_evaluator()
+        evaluators = copy.copy(first_evaluator.get_social_eval_list())
+        evaluators += first_evaluator.get_ta_eval_list()
+        evaluators += first_evaluator.get_real_time_eval_list()
+        for evaluator in evaluators:
+            config_eval.append(evaluator.get_name())
+        return config_eval
+
+    def _init_bot_id(self):
+        answer = requests.get(f"{METRICS_URL}{METRICS_ROUTE_GEN_BOT_ID}", headers=self._headers)
```
Comment (Werdor) [inline tools/metrics_manager.py]: There is no way to use requests or equivalent with async ?

Comment (GuillaumeDSM) [reply tools/metrics_manager.py]: There should be, I need to look into it

File: core/octobot.py (hunk 1 above)
Comment (Werdor) [reply core/octobot.py]: Ok :+1:

**Pull request merged**